api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -71,4 +70,55 @@
 
     Long getUserCount();
   }
+
+  @Query(
+      "SELECT\n"
```

Comment:
I had the query string as a constant in the projection class. That approach is certainly tidier, but you lose IntelliJ's highlighting that understands this SQL.